modules/xfem/src/efa/EFAElement3D.C

```diff
@@ -2050,6 +2050,21 @@ EFAElement3D::addFaceEdgeCut(unsigned int face_id,
     if (cut_edge->numEmbeddedNodes() > 2)
       EFAError("element edge can't have >2 embedded nodes");
 
+    unsigned int FragFaceID = 99999, FragFaceEdgeID = 99999;
+    if (getFragmentFaceEdgeID(face_id, edge_id, FragFaceID, FragFaceEdgeID))
+    {
+      EFAEdge * elem_edge = _faces[face_id]->getEdge(edge_id);
+      EFAEdge * frag_edge = getFragmentFace(0, FragFaceID)->getEdge(FragFaceEdgeID);
+      double xi[2] = {-1.0, -1.0}; // relative coords of two frag edge nodes
+      xi[0] = elem_edge->distanceFromNode1(frag_edge->getNode(0));
+      xi[1] = elem_edge->distanceFromNode1(frag_edge->getNode(1));
+      double frag_pos = (position - xi[0]) / (xi[1] - xi[0]);
```

Comment:
would xi[1] - xi[0] = 0?

Reply:
My understanding, which could be wrong, is that this means the frag-edge has zero length.  This is a very special case that I can look into.  I just made another push without changing anything here.  I will look into this later and if needed, I would make another PR with a new Issue since this is not the only occurrence of this calculations.

Reply:
I'm not sure whether we would ever allow xi[0]=xi[1]. I'm OK deferring this to a later commit, but we should look into this. The same sort of thing might show up in other places as well.